### PR TITLE
Add tools subdir containing vmlist.py script

### PR DESCRIPTION
**Pull request by dmick**
Signed-off-by: Dan Mick <dan.mick@redhat.com>

### CODE REVIEW

**Comment by ktdreyer** [comment]
LGTM

**Comment by dmick** [comment]
looking over it, doesn't make a lot of sense to allow one worker to be called with no outputfile but the other one to effectively require it...but, meh.

File: tools/vmlist.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+import sys
+import tempfile
+import textwrap
+
+CACHEFILE = "~/.vmlist.cache"
+CONFFILE = "~/.vmlist.conf"
+
+# mira004.front.sepia.ceph.com is dead
+
+VMMACHINES = textwrap.dedent('''\
```
Comment (zmc) [inline tools/vmlist.py]: How about VM_HOSTS? Also, how about letting it be overridden by setting an env variable of the same name?

**Comment by zmc** [comment]
Curious: why write and re-read outputfiles and not simply use buffers?

File: tools/vmlist.py (hunk 1 above)
Comment (dmick) [inline tools/vmlist.py]: Name change is fine with me; env vars: well, you can set it in the .conf file too;  I guess we could define an order env -> conf -> default.  You think others of the things should also be in the env, or just the hosts?  (note also the -m switch)

**Comment by dmick** [comment]
outputfiles because multiple processes.  I guess there *are* other methods...Queue and Pipe...but it was not at all clear how to use those, to me, and I wanted to get something working.  I would rather use memory than the filesystem though.

**Comment by dmick** [comment]
Just a poke to get this moving.  Anything else we wanna do to it?

**Pull request merged**